<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>590</width>
      <height>850</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:white;
border-color:black;
border-width:1px;
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="image_name">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:white;
border-color:black;
border-style:plane;
border-width:1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image_data">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:white;
border-color:black;
border-style:plane;
border-width:1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image_targa">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:white;
border-color:black;
border-style:plane;
border-width:1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_name">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>60</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:white;
border-color:black;
border-style:plane;
border-width:1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_data">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>190</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:white;
border-color:black;
border-style:plane;
border-width:1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_targa">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>310</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:white;
border-color:black;
border-style:plane;
border-width:1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="readButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>680</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(76, 133, 255);
color:white;</string>
    </property>
    <property name="text">
     <string>READ</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_cf">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>410</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:white;
border-color:black;
border-style:plane;
border-width:1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_cf">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>420</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:white;
border-color:black;
border-style:plane;
border-width:1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_via">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>530</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:white;
border-color:black;
border-style:plane;
border-width:1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image_via">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>520</y>
      <width>250</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:white;
border-color:black;
border-style:plane;
border-width:1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
